docs/source/features/disagg-serving.md

```
@@ -0,0 +1,260 @@
+# Disaggregated Serving
+
+- [Motivation](#Motivation)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
Expected: #motivation; Actual: #Motivation

(MD051, link-fragments)